trl/trainer/ppo_trainer.py

```diff
@@ -390,6 +406,7 @@ def step(
             stats = self.gather_stats(stats)
         stats = stats_to_np(stats)
         timing["time/ppo/calc_stats"] = time.time() - t
+        stats["learning_rate"] = self.optimizer.param_groups[0]["lr"]
```

Comment:
what do you think about adding a prefix which means it gets ordered in wandb logs?

```suggestion
        stats["ppo/learning_rate"] = self.optimizer.param_groups[0]["lr"]
```